pkg/trace/api/otlp.go

```diff
@@ -509,6 +509,13 @@ func (o *OTLPReceiver) convertSpan(rattr map[string]string, lib pcommon.Instrume
 				name = "opentelemetry." + name
 			}
 		}
+
+		normalized, err := traceutil.NormalizeName(name)
```

Comment:
I'm not convinced by this change. The remapping feature should remap the OpenTelemetry name to a new Datadog name, which gets normalised later on, if it has to. Doing this here will just create confusion and potential issues later on.

In my opinion, the current behaviour is better and the span name remappings should refer to the actual OpenTelemetry name, and not a Datadog normalized version of it.

Reply:
@gbbr 
Thank you for reviewing and the comment.

I got your point and yes, you're right.
Even though I created this pull request for the compatibility, I also think the current behavior is ideal.

So I'm happy to close this pull request, however it would be great if you mention the breaking change on the remapping feature and update [the `span_name_remappings ` example](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/143cafc3c376c627058209334d50ecf97ed7b205/exporter/datadogexporter/example/config.yaml#L199-L202) on in https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/9693

Thank you!

Reply:
Agreed. Added here: https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/9693/commits/2fa0ec9ca88c58968dab0d6bb8da0f9e58f3f6bd

Reply:
Thank you for adding to CHANGELOG.

If possible, I think it's better to update the example here as well:
https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/9693/files#diff-8c91465b7f1defd1410e697b702a817163bc88cee637ace3f202bb7f83321cb6L202

I'm closing this pull request.
Thank you!